paddle/fluid/framework/data_set.h

```diff
@@ -94,6 +96,9 @@ class Dataset {
   virtual void LocalShuffle() = 0;
   // global shuffle data
   virtual void GlobalShuffle() = 0;
+  // for slots shuffle
+  virtual void SlotsShuffle(const std::set<uint16_t>& slots_to_replace) = 0;
```

Comment:
why it's always int，what if user use string as a slot name, such as "myslot"

Reply:
to do

Reply:
changed to std::string